phpstan.neon.dist

```diff
@@ -7,8 +10,12 @@ parameters:
         - tests
     excludePaths:
         - tests/Helpers/LegacyLogger.php
-
     ignoreErrors:
-        # Ignore some irrelevant errors in test files
-        - '~Method Composer\\XdebugHandler\\Tests\\[^:]+::test[^(]+\(\) has no return type specified.~'
-        - '~Method Composer\\XdebugHandler\\Tests\\[^:]+::\w+?Provider\(\) has no return type specified.~'
+        # strict-rules errors
+        - message: "#^Call to function is_string\\(\\) with string will always evaluate to true.#"
```

Comment:
A better fix than ignoring this error is to set the parameter `treatPhpDocTypesAsCertain: false` so that phpstan does not complain when checking types that are only defined in phpdoc and not enforced by a type check.